1-mandatory/1-writers.js

```diff
@@ -63,18 +69,36 @@ Exercise 2:
 
 function logDeadWritersInTheirForties() {
   // write your code here
+  for (let i = 0; i < writers.length; i++) {
+    let writer = writers[i];
+    {
+      if (writer.age >= 40 && writer.age < 50 && writer.alive == false) {
+        console.log(
+          `Writer ${writer.firstName} ${writer.lastName} died at ${writer.age} years old.`
+        );
+      }
+    }
+  }
 }
-
 /*
 Exercise 3:
   Only `console.log()` out alive writers who are in their 40s (meaning between 40 and 49):
   "Hi, my name is {firstName} {lastName}. I am {age} years old."
 */
 
-function logAliveWritersInTheirForties() {
+function logAliveWritersInTheirForties(array) {
   // write your code here
+  for (let i = 0; i < writers.length; i++) {
+    let writer = writers[i];
+    {
+      if (writer.age >= 40 && writer.age < 50 && writer.alive !== false) {
+        console.log(
+          `Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old.`
+        );
+      }
+    }
```

Comment:
These "for" loops look good.

The curly brackets around the "if" blocks are not needed. For example in this loop on line 93 and 98. This is not an error, more just question of code styling. The if statement already defines its own block.

Personally, I don't have an issue using the traditional for loop (like you did), although some people might say you could try using
`writers.forEach(...`.